packages/gatsby-theme-newrelic/README.md

```diff
@@ -1923,6 +1966,33 @@ import { NewRelicLogo } from '@newrelic/gatsby-theme-newrelic';
 <NewRelicLogo />
 ```
 
+### Overlay
+
+Used as a container to display a any component passed as children on top of a page while stopping scrolling on the page itself.
+
+```js
+import { Overlay } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Props**
+
+| Prop             | Type     | Required | Default | Description                                                                  |
+| ---------------- | -------- | -------- | ------- | ---------------------------------------------------------------------------- |
+| `className`      | string   | no       |         | Additional `className` for the component.                                    |
+| `children`       | node     | no       |         | Content to be displayed in the `Overlay` component                           |
+| `onCloseOverlay` | function | no       |         | Additional function that can be run as an onClick when closing the `Overlay` |
+| `isOpen`         | boolean  | no       | false   | Boolean to determine if the `Overlay` is displayed                           |
```

Comment:
According to the [propTypes](https://github.com/newrelic/gatsby-theme-newrelic/blob/main/packages/gatsby-theme-newrelic/src/components/Overlay.js#L112-L114) these props are required.
```suggestion
| `children`       | node     | yes        |         | Content to be displayed in the `Overlay` component                           |
| `onCloseOverlay` | function | yes        |         | Additional function that can be run as an onClick when closing the `Overlay` |
| `isOpen`         | boolean  | yes        | false   | Boolean to determine if the `Overlay` is displayed                           |
```